dask/delayed.py

```diff
@@ -46,6 +47,16 @@ def finalize(collection):
     return Delayed(name, graph)
 
 
+def is_namedtuple_instance(obj: Any) -> bool:
+    "Returns True if obj is an instance of a namedtuple."
+    return (
+        isinstance(obj, tuple)
+        and hasattr(obj, "_asdict")
+        and hasattr(obj, "_fields")
+        and hasattr(obj, "_replace")
+    )
```

Comment:
For reviewers: this is safer than it looks, as these are first-class attributes on the `namedtuple` API, just prefixed with underscores to avoid name collisions with field names: https://docs.python.org/3/library/collections.html#namedtuple-factory-function-for-tuples-with-named-fields

So I think this check is a decent one (though it might be good to add a comment for why these attribute checks are okay)

Reply:
Thanks for pointing that out @ian-r-rose. I think a comment here would be nice to help avoid confusion / concern.

Reply:
Comment added